ApollosProject/apollos-apps
# fix(community): remove spacing at bottom of activity feed in dark mode

## ✏️ Description
- removes spacing at very bottom of activity feed for community tab when device is in dark mode


## 🔬 To Test
1. Set device to dark mode
2. Go to Community Tab
3. Scroll all the way down to the bottom of the activity feed
4. You should not see any whitespace at the bottom

## 📸 Screenshots
![image](https://user-images.githubusercontent.com/7557597/215597377-9c4663de-0d94-4da1-9100-ce93063c271b.png)



## ⚠️ To-do Before Merge



## 📝 Checklist
